test/syncenginetestutils.cpp

```diff
@@ -589,6 +590,7 @@ void FakePutMultiFileReply::respond()
         QJsonObject fileInfoReply;
         fileInfoReply.insert("error", QStringLiteral("false"));
         fileInfoReply.insert("etag", QLatin1String{fileInfo->etag});
+        fileInfoReply.insert("x-oc-mtime", QLatin1String("accepted"));
```

Comment:
why do you set it ?
is it set by  real server ?

Reply:
hmm, this does not get set by the server indeed

will remove it